### ISSUE TITLE
[windows] jmxfetch not shutting down when the agent shuts down

### ISSUE THREAD

**Issue by clofresh** [DataDog/dd-agent#886]
From a customer:

```
The way we were doing the install was run the windows installer, shut down agent, install my jmx config files and restarting
When I did service stop, it did not stop jmxfetch, so I had multiple jmxfetches ruinning
```

**Comment by MartinDatadog** [comment]
Reported by bmontague from org 4456
https://datadog.zendesk.com/agent/tickets/25726
With latest version of the datadog-agent (5.3.1.), restarting the agent creates a new jmxfetch process and the old ones are not killed. I asked for more info.

**Comment by MartinDatadog** [comment]
We don't have logs for now, but the customer will reach out with logs if it happens again. @remh

**Comment by MartinDatadog** [comment]
Full logs from customer regarding this issue: https://datadog.zendesk.com/agent/tickets/26788 @remh 
Agent version 5.3.1 (windows)

**Comment by MartinDatadog** [comment]
Another customer impacted by this issue:
org 19892 https://datadog.zendesk.com/agent/tickets/28711
Every time they restart the datadog-agent a new java.exe process is created, but the old jmxfetch process doesn't get terminated.

**Comment by MartinDatadog** [comment]
Upgrading to agent 5.4 seems to have fixed their problem. @remh Do you want to close this issue?

**Comment by tmichelet** [comment]
@MartinDatadog closing, this issue has been fixed in the 5.4 release
